File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex18.java
//Importando Scanner
import java.util.Scanner;

public class Ex18 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	float tempC, tempF;
	
	System.out.println("Temperatura °C --> °F \n\n");
	
	//Entrada
	System.out.print("Insira a tempratura em °C: ");
	tempC = ent.nextFloat();
	
	//Processamento
	tempF = (tempC * 9 / 5) + 32;
	
	//Saída
	System.out.printf("Temperatura °F: %.1f", tempF);
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex18.c
#include <stdio.h>

main()
{
    //vars
    float celsius, farenheit;

    printf("\t\t\tConversao Celsius --> Farenheit\n\n");

    //Entrada
    printf("Insira a temperatura em celsius: ");
    scanf("%f",&celsius);

    //Processamento
    farenheit = (celsius * 9 + 160) / 5;

    //Saída
    printf("Temperatura em Farenheit: %.1f", farenheit);

}
